src/commands/find.ts

```diff
@@ -0,0 +1,15 @@
+import * as vscode from "vscode";
+import { TextEditor } from "vscode";
+import { EmacsCommand } from ".";
+
+export const findCommandIds = [
```

Comment:
I think this `findCommandIds` is not necessary and in `extension.ts` you can directly do
```
    registerEmulatorCommand(`emacs-mcx.find`, emulator => {
      emulator.runCommand("find");
    });
```

`moveCommandIds` in `commands/move.ts` is not a general rule... sorry it's confusing.